Formula/rocksdb.rb

```diff
@@ -104,7 +111,8 @@ def install
                                 "-L#{lib}", "-lrocksdb",
                                 "-L#{Formula["snappy"].opt_lib}", "-lsnappy",
                                 "-L#{Formula["lz4"].opt_lib}", "-llz4",
-                                "-L#{Formula["zstd"].opt_lib}", "-lzstd"
+                                "-L#{Formula["zstd"].opt_lib}", "-lzstd",
+                                "-I#{include}"
```

Comment:
That looks like it's a bug since that should already be the case.

Reply:
Oh okay, I'll remove that. Should we remove `"-L#{lib}"` too?

Reply:
Seems to build just fine without `"-L#{lib}"`, so I'm guessing it's included by default in `ENV.cxx`.

Reply:
Have a look at my latest push, providing it passes.